src/Compilers/CSharp/Test/Semantic/Semantics/OverloadResolutionTests.cs

```diff
@@ -9456,6 +9456,420 @@ public void PassingInArgumentsOverloadedOnIn()
         {
             var code = @"
 public static class Program
+{
+    public static void Method(in int x)
+    {
+        System.Console.WriteLine(""in: "" + x);
+    }
+
+    public static void Method(int x)
+    {
+        System.Console.WriteLine(""val: "" + x);
+    }
+
+    public static void Main()
+    {
+        int x = 5;
+        Method(in x);
+        Method(x);
+        Method(5);
+    }
+}";
+
+            CompileAndVerify(code, expectedOutput: @"
+in: 5
+val: 5
+val: 5
+");
+        }
+
+        [Fact]
+        public void PassingInArgumentsOverloadedOnIn_BinaryOperators()
```

Comment:
>PassingInArgumentsOverloadedOnIn_BinaryOperators [](start = 20, length = 48)

It would be good to test similar scenario, but when methods are declared in a different order.